container.go

```diff
@@ -29,7 +29,8 @@ import (
 // Process gathers data related to a container process.
 type Process struct {
 	Token     string
-	Pid       int
+	HostPID   int
+	GuestPID  int
```

Comment:
And this is the guest kernel PID (from the root namespace perspective) of the workload? Given the potential for confusion, I think we should add comments explaining which "level" all the new PID fields refer to.